### PR TITLE
Fix: Updated formatting of text throughout the app

### PR DESCRIPTION
**Pull request by Mikehrn**
Had some time while waiting for a pipeline to finish so I took a look at the text formatting because it was a bit inconsistent. 

Now we've decided to update the text formatting (mainly capitalization) I went through the app and adjusted the copy to consistently use the new way of formatting (only capitalizing the first letter of a sentence) and fixed some typos. Should fix the text formatting in most of the app. Only touches copy. 

Ignored the settings modal as I've adjusted the text formatting in the other PR

### CODE REVIEW

File: packages/frontend-2/components/auth/LoginPanel.vue [hunk 1]
```diff
@@ -63,7 +63,7 @@ const props = withDefaults(
   }>(),
   {
     dialogMode: false,
-    title: 'Speckle Login',
+    title: 'Speckle login',
     subtitle: 'Connectivity, Collaboration and Automation for 3D'
```
Comment (Mikehrn) [inline packages/frontend-2/components/auth/LoginPanel.vue:67]: Wasn't sure if we wanted to change this tagline, or we use it across different places with this formatting

Comment (benjaminvo) [reply packages/frontend-2/components/auth/LoginPanel.vue:67]: Yeah let's just keep this as is for now

File: packages/frontend-2/pages/downloads.vue [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+        <h5 class="h4 font-bold">Connector downloads</h5>
         <div class="text-sm text-foreground-2 max-w- max-w-sm">
           Most of our connectors are available through Speckle Manager. You can also
           direct download the individual installers below.
         </div>
       </div>
       <div>
         <FormButton size="lg" full-width @click="showManagerDownloadDialog = true">
-          Download Manager
+          Download manager
```
Comment (benjaminvo) [inline packages/frontend-2/pages/downloads.vue:18]:
```suggestion
          Download Manager
```
Should be uppercase. Manager is a product name, like "Speckle" or "Revit".

**Review by benjaminvo — CHANGES_REQUESTED** [review]
Veeery nice. I noticed one tiny thing!

File: packages/frontend-2/pages/downloads.vue (hunk 2 above)
Comment (Mikehrn) [reply packages/frontend-2/pages/downloads.vue]: Got it! Updated :)

**Review by benjaminvo — APPROVED** [review]

**Pull request merged**